assets/airports/mdsd.json

```diff
@@ -0,0 +1,5081 @@
+                ["N18.400760700", "W70.049377000"],
+                ["N18.401628800", "W70.049448200"],
+                ["N18.402499200", "W70.049472000"],
+                ["N18.403369600", "W70.049448400"],
+                ["N18.404237700", "W70.049377300"],
+                ["N18.405101300", "W70.049259000"],
+                ["N18.405957900", "W70.049093800"],
+                ["N18.406805500", "W70.048882000"],
+                ["N18.407641500", "W70.048624200"],
+                ["N18.408464000", "W70.048321100"],
+                ["N18.409270600", "W70.047973300"],
+                ["N18.410059200", "W70.047581700"],
+                ["N18.410827500", "W70.047147400"],
+                ["N18.411573700", "W70.046671400"],
+                ["N18.412295400", "W70.046154900"],
+                ["N18.412990900", "W70.045599300"],
+                ["N18.413658100", "W70.045006100"],
+                ["N18.414295200", "W70.044376600"],
+                ["N18.414900300", "W70.043712600"],
+                ["N18.415471800", "W70.043016000"],
+                ["N18.416008000", "W70.042288400"],
+                ["N18.416507300", "W70.041532000"],
+                ["N18.416968300", "W70.040748800"],
+                ["N18.417389600", "W70.039940700"],
+                ["N18.417770100", "W70.039110300"],
+                ["N18.418108500", "W70.038259500"],
+                ["N18.418403900", "W70.037391000"],
+                ["N18.418655400", "W70.036507000"],
+                ["N18.418862200", "W70.035610100"],
+                ["N18.419023700", "W70.034702800"],
+                ["N18.419139400", "W70.033787800"],
+                ["N18.419208900", "W70.032867400"],
+                ["N18.419232100", "W70.031944400"]
+            ]
+        }
+    ],
+    "runways":[
+        {
+            "name": ["17", "35"],
+            "end": [
+                ["N18d26m38.26", "W69d40m27.04", "39ft"],
+                ["N18d24m55.31", "W69d39m49.22", "58ft"]
+            ],
+            "ils": [true, false]
+        }
+    ],
+    "sids": {
+        "AVRES": {
+            "icao": "AVRES",
+            "name": "Avres",
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", ["SD134", "A40-"]],
+                "35": ["^SD140", "SD144", "SD145", ["SD146", "A30"], ["SD147", "A40+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "AVRES": ["AVRES"]
+            },
+            "draw": [
+                ["SD130", "SD134", "AVRES"],
+                ["SD140", "SD144", "SD145", "SD146", "SD147", "AVRES"]
+            ]
+        },
+        "AVRES1": {
+            "icao": "AVRES1",
+            "name": "Avres One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "CDO"],
+            "exitPoints": {
+                "AVRES": ["AVRES"]
+            },
+            "draw": [
+                ["CDO", "AVRES"]
```

Comment:
Drawing the line from PITOX to CDO makes little sense, since PITOX is a fly-over fix, the line would be sort of misleading